src/openms/include/OpenMS/METADATA/PeptideEvidence.h

```diff
@@ -55,7 +55,7 @@ namespace OpenMS
 
     // Note: we use 0 as position of the N-terminus while e.g. mzTab or other formats start counting at 1.
     static const int N_TERMINAL_POSITION;
-    static const char UNKNOWN_AA; // PeptideEvidence::UNKNOWN_AA = ' ';
+    static const char UNKNOWN_AA; // PeptideEvidence::UNKNOWN_AA = 'X';
```

Comment:
XML stores lists as space separated and we now need non-whitespace characters for the list entries.